### DISCUSSION TITLE
askpdf没有办法知道这篇文章的内容

### DISCUSSION THREAD

**Discussion by lokkkkyi** [MuiseDestiny/zotero-gpt#289] [Q&A]
如下图，askpdf功能没有办法查看里面的内容，可以帮忙解决一下吗🥺
<img width="813" alt="截屏2024-01-31 下午2 18 37" src="https://github.com/MuiseDestiny/zotero-gpt/assets/157364259/3ea8321a-5409-465d-805d-e8d84d978a55">

**Comment by wangeas** [comment]
加一

**Comment by wangeas** [comment]
<img width="335" alt="截屏2024-05-01 20 58 06" src="https://github.com/MuiseDestiny/zotero-gpt/assets/168623155/acedd100-2090-421e-802d-186702afd05f">
<img width="343" alt="截屏2024-05-01 20 48 54" src="https://github.com/MuiseDestiny/zotero-gpt/assets/168623155/3377b49f-16e9-470e-a71b-f1bbb64f23a5">
<img width="252" alt="截屏2024-05-01 20 49 05" src="https://github.com/MuiseDestiny/zotero-gpt/assets/168623155/ba1344a9-c52c-47e4-93fe-e5ec5b8f3778">

**Reply by wangeas** [reply]
似乎是阅读不了pdf
